### PR TITLE
Doc: Clusters from Scratch: convert to sphinx

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: doc/sphinx/Clusters_from_Scratch/ap-configuration.rst [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+    Daemon Status:
+      corosync: active/disabled
+      pacemaker: active/disabled
+      pcsd: active/enabled
+
+::
+
+    [root@pcmk-1 ~]# pcs cluster cib --config
+
+.. code:: xml
```
Comment (kgaillot) [inline doc/sphinx/Clusters_from_Scratch/ap-configuration.rst:80]: code-block (unless that's another difference between versions?)

File: doc/sphinx/Clusters_from_Scratch/installation.rst [hunk 2]
```diff
@@ -0,0 +1,409 @@
+Installation
+------------
+
+Install |CFS_DISTRO| |CFS_DISTRO_VER|
+################################################################################################
+
+Boot the Install Image
+______________________
+
+Download the 4GB `|CFS_DISTRO| |CFS_DISTRO_VER| DVD ISO <http://isoredirect.centos.org/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1804.iso>`_.
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/installation.rst:10]: Indeed it does.  That's too bad.  Fixed.

File: doc/sphinx/Clusters_from_Scratch/cluster-setup.rst [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+
+    [root@pcmk-1 ~]# pcs cluster auth pcmk-1 pcmk-2
+    Username: hacluster
+    Password:
+    pcmk-2: Authorized
+    pcmk-1: Authorized
+
+.. NOTE::
+
+    In Fedora 29 and CentOS 8.0, the command has been changed to `pcs host auth`:
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:147]: Fixed.

File: doc/sphinx/Clusters_from_Scratch/verification.rst [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    Other operating systems may report startup errors in other locations,
+    for example ``/var/log/messages``.
+
+Repeat these checks on the other node. The results should be the same.
+
+Explore the Existing Configuration
+##################################
+
+For those who are not of afraid of XML, you can see the raw cluster
+configuration and status by using the `pcs cluster cib` command.
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/verification.rst:158]: Fixed.

File: doc/sphinx/Clusters_from_Scratch/apache.rst [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+
+There are always times when an administrator needs to override the
+cluster and force resources to move to a specific location. In this example,
+we will force the WebSite to move to pcmk-1.
+
+We will use the **pcs resource move** command to create a temporary constraint
+with a score of INFINITY. While we could update our existing constraint,
+using **move** allows to easily get rid of the temporary constraint later.
+If desired, we could even give a lifetime for the constraint, so it would
+expire automatically -- but we don't that in this example.
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/apache.rst:336]: Fixed.

File: doc/sphinx/Clusters_from_Scratch/ap-configuration.rst (hunk 1 above)
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/ap-configuration.rst:80]: No, just a thinko from running through all these files in a day.

File: doc/sphinx/Clusters_from_Scratch/cluster-setup.rst [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+
+Configure the Cluster Software
+##############################
+
+Allow cluster services through firewall
+_______________________________________
+
+On each node, allow cluster-related services through the local firewall:
+
+::
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:61]: Do that everywhere in this PR?  If so, I'll also need to make that change in the Pacemaker_Remote document too.

File: doc/sphinx/Clusters_from_Scratch/cluster-setup.rst [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+and some other command-line tools that will make our lives easier:
+
+::
+
+    # yum install -y pacemaker pcs psmisc policycoreutils-python
+
+.. IMPORTANT::
+
+    This document will show commands that need to be executed on both nodes
+    with a simple ``#`` prompt. Be sure to run them on each node individually.
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:45]: It looks like this to me:

Screenshot from 2020-05-01 09-56-15

File: doc/sphinx/Clusters_from_Scratch/installation.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/installation.rst:10]: Alt text doesn't support substitutions either:
```
<a class="reference internal image-reference" href="_images/Editing-eth0.png"><img alt="|CFS_DISTRO| |CFS_DISTRO_VER| Editing eth0" src="_images/Editing-eth0.png" style="width: 1024.0px; height: 800.0px;" />
```

File: doc/sphinx/Clusters_from_Scratch/installation.rst [hunk 8]
```diff
@@ -0,0 +1,409 @@
...
+boot a physical server from that.
+
+After starting the installation, select your language and keyboard layout at
+the welcome screen.
+
+.. figure:: ../../shared/en-US/images/Welcome.png
+    :scale: 100%
+    :width: 100%
+    :align: center
+    :alt: Welcome to |CFS_DISTRO| |CFS_DISTRO_VER|
```
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/installation.rst:21]: Scaling to about 80% looks pretty good here.

File: doc/sphinx/Clusters_from_Scratch/cluster-setup.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:61]: Done.

[the remark this answers is not in the document]
Comment (kgaillot) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:61]: Just where it highlights things wrong would be sufficient, but everywhere doesn't hurt

File: doc/sphinx/Clusters_from_Scratch/cluster-setup.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (kgaillot) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:45]: I suppose that's fine then. I don't want to potentially make it suboptimal for future versions just to support old versions. We'll just need to make sure we generate the online docs using a recent version.

[the remark this answers is not in the document]
Comment (clumens) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:45]: We could always add the newer CSS for styling it into pacemaker.css.

[the remark this answers is not in the document]
Comment (kgaillot) [reply doc/sphinx/Clusters_from_Scratch/cluster-setup.rst:45]: Right, that's what I meant by making it suboptimal for future versions -- if they improve the default styling, we'd still be stuck with whatever's there now. The issue doesn't make it unreadable, just less emphasized, so I'd rather let time take care of it.

**Comment by clumens** [comment]
I hadn't actually pushed the latest version of this PR yet.  I was waiting to see what would happen with the information styling.

**Pull request merged**